### ISSUE TITLE
Air 2S SDK Support

### ISSUE THREAD

**Issue by checa14** [dji-sdk/Mobile-SDK-iOS#456]
Please, support the Air 2S as soon as possible. It is a shame that the Mavic Air 2 and the Mavic Mini are supported while better, more modern and expensive drones like the Air 2S or the Mini 2 are not.

I understand that they support the Mavic 2 Enterprise Advanced sooner, but the Air 2S should be on the road map for June as well.

**Comment by dji-dev** [comment]
Agent comment from Luce Luo in <a href='https://djisdksupport.zendesk.com/agent/tickets/46921'>Zendesk ticket #46921</a>:<br><br>Dear Customer,

Thank you for contacting DJI.
Sorry that there is no plan to support the Air 2S, Mini2.
Please attention to the SDK ROADMAP if any news in the future.
https://developer.dji.com/user/roadmap/#msdk

Thanks,

Luce Luo
DJI Developer Support

**Comment by Mattmst-ai** [comment]
Good To know so i don't buy the dji Air 2s, and mini2

**Comment by albertoxamin** [comment]
👎🏻 👎🏻 👎🏻 👎🏻 👎🏻 👎🏻 👎🏻 👎🏻

**Comment by DaveBGld** [comment]
I wonder if DJI is violating US antitrust law and hence exposing itself to a major class action lawsuit...???

**Comment by nicksargeant** [comment]
I will be returning the Mini 2 drone I just purchased. Very disappointing.

**Comment by Eddie-Hartman** [comment]
@dji-dev If there is not going to be any SDK support for the Air 2S, it seems like you should close this ticket.

I would hope for SDK support, but leaving the ticket open continues to give me hope.

**Comment by LoveYou69** [comment]
Please, support the Air 2S as soon as possible.

Thanks :)

**Comment by EdMat1844** [comment]
> Please, support the Air 2S as soon as possible. It is a shame that the Mavic Air 2 and the Mavic Mini are supported while better, more modern and expensive drones like the Air 2S or the Mini 2 are not.
> 
> I understand that they support the Mavic 2 Enterprise Advanced sooner, but the Air 2S should be on the road map for June as well.

I second this user.

**Comment by NullP0interEx** [comment]
Please add SDK support for this drone asap

**Comment by jakevis** [comment]
I think I'll be demanding a refund I'd the SDK isn't opened up... Totally unacceptable.

**Comment by LoveYou69** [comment]
up

**Comment by NullP0interEx** [comment]
Yes, I don't think that's acceptable either.  I have high expectations of premium manufacturers like DJI, but apparently the quality of the products and the service is not much better than the next best Chinese from Shenzhen.  I bought the drone for experiments at university.  I can now forget that because I don't have the SDK.

**Comment by albertoxamin** [comment]
A YouTuber reported that dji told him the sdk support will come with v5.0. But we still need to wait long for that release.

**Comment by MikesTooLz** [comment]
It just boggles my mind as to why SDK support is not made available before the drone is out for sale. These are things users expect to have when purchasing any DJI drone.

**Comment by StanLee4456** [comment]
Hilarious that you don't support the SDK months after the drone is released.
Hilarious that you "have no plans to support the SDK".

Combined - that's just pretty sad and pathetic.

**Comment by rlesac** [comment]
This is unacceptable. I too will be returning the Air 2S.

**Comment by LoveYou69** [comment]
Any news?

**Comment by Studentdroner** [comment]
@dji-dev 

Was really looking forward to utilizing the air 2s for photography and mapping. This would have been my first drone and I am very disappointed that DJI made this information extremely hard to find as there is little discussion about waypoints and mapping capabilities.

**Comment by LoveYou69** [comment]
> 
> 
> @dji-dev
> 
> Was really looking forward to utilizing the air 2s for photography and mapping. This would have been my first drone and I am very disappointed that DJI made this information extremely hard to find as there is little discussion about waypoints and mapping capabilities.

Same here :(

**Comment by NullP0interEx** [comment]
And here ….

**Comment by sschmaus** [comment]
same story here...

**Comment by powolnymarcel** [comment]
Support is coming guys 

https://github.com/dji-sdk/Mobile-SDK-Android/issues/988#issuecomment-888083667

**Comment by NullP0interEx** [comment]
Great 👍

**Comment by powolnymarcel** [comment]
Got an email from DJI where they say it will be supported in 
"we plan to support Air 2S in MSDK 4.16"

**Comment by felixsc1** [comment]
It would allow for so many more stunning and smooth videos on youtube, instagram etc. 
I don't get why they would pass this free advertisement.  The competition (xiaomi fimi drones for example) have this feature.

**Comment by LoveYou69** [comment]
Any news?

**Comment by alcantarajimmy** [comment]
Hello forum members. I invite you to sign and share the request for DJI to support the Air 2s and Mini2 SDK. 
https://www.change.org/sdkforair2s
Together, we consumers can generate changes.

**Comment by alcantarajimmy** [comment]
> @ dji-dev
> 
> Tenía muchas ganas de utilizar los air 2 para fotografías y mapas. Este habría sido mi primer dron y estoy muy decepcionado de que DJI haya hecho que esta información sea extremadamente difícil de encontrar, ya que hay poca discusión sobre los puntos de referencia y las capacidades de mapeo.

The same happened to me. So I have started a request for DJI to support SDK for Air 2s and mini2 asap. I invite you to sign and share.

https://www.change.org/sdkforair2s

**Comment by powolnymarcel** [comment]
Support added :) 
Android Mobile SDK 4.16
https://developer.dji.com/

**Comment by alcantarajimmy** [comment]
thanks dji team!!

**Comment by dj3183** [comment]
I can’t believe that this air 2S does not have video waypoints. I am taking mine back as well unless you update your firmware and app.

**Comment by JmAguirreL** [comment]
> Agent comment from Luce Luo in [Zendesk ticket #46921](https://djisdksupport.zendesk.com/agent/tickets/46921):Dear Customer,
> 
> Thank you for contacting DJI. Sorry that there is no plan to support the Air 2S, Mini2. Please attention to the SDK ROADMAP if any news in the future. https://developer.dji.com/user/roadmap/#msdk
> 
> Thanks,
> 
> Luce Luo DJI Developer Support

Any news?
Thank you Luceo in advanced...

**Comment by godfreynolan** [comment]
According to the roadmap, support for Air 2S and Mini2 was added in March for iOS, see SDK version 4.16.
